script-generator/.gitignore

```diff
@@ -12,4 +13,4 @@
 =======
 .classpath
 .project
->>>>>>> dev
+>>>>>>> dev
```

Comment:
Please remove these changes.

Reply:
@hank121314 I have removed unrelated change you mention, please check it

Reply:
Seems still exist in the latest commit.